RELEASES.md

```diff
@@ -1,3 +1,10 @@
+# Enso Next
+
+## Miscellaneous
+
+- Adding a pipeline for automatic nightly builds
+  ([#1689](https://github.com/enso-org/enso/pull/1689)).
```

Comment:
I'd approach this from the perspective of mentioning that we now have nightly builds, and the limitations on them. These changelogs _are_ user-facing to an extent.